spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitCommonProperties.java

```diff
@@ -90,4 +177,141 @@ public void setBindingRoutingKey(String routingKey) {
 		this.bindingRoutingKey = routingKey;
 	}
 
+	public Integer getTtl() {
+		return this.ttl;
+	}
+
+	public void setTtl(Integer ttl) {
+		this.ttl = ttl;
+	}
+
+	public Integer getExpires() {
+		return this.expires;
+	}
+
+	public void setExpires(Integer expires) {
+		this.expires = expires;
+	}
+
+	public Integer getMaxLength() {
+		return this.maxLength;
+	}
+
+	public void setMaxLength(Integer maxLength) {
+		this.maxLength = maxLength;
+	}
+
+	public Integer getMaxLengthBytes() {
+		return this.maxLengthBytes;
+	}
+
+	public void setMaxLengthBytes(Integer maxLengthBytes) {
+		this.maxLengthBytes = maxLengthBytes;
+	}
+
+	public Integer getMaxPriority() {
+		return this.maxPriority;
+	}
+
+	@Range(min = 0, max = 255)
+	public void setMaxPriority(Integer maxPriority) {
```

Comment:
M-m-m. Shouldn't this validation annotation be on the getter ?
At least that is what I see in the app starters...